src/ts/ConvivaAnalytics.ts

```diff
@@ -205,16 +204,24 @@ export class ConvivaAnalytics {
       'preload': PlayerConfigHelper.getPreloadConfig(this.player) + '',
       ...this.config.customTags,
     };
+
+    // also include dynamic content metadata at initial creation
+    this.buildDynamicContentMetadata();
   }
 
   /**
    * Update contentMetadata which are allowed during the session
    */
+  private buildDynamicContentMetadata() {
```

Comment:
why is this named `dynamicContentMetadata`.

Reply:
hm. was the best wording i found for content metadata that can be changed over time.